File: PYTHON/haris/binomial_coefficient.py
def binomial(x , y):
    from math import factorial as fac

    # print("\nBinomial Coefficient Calculator. \n\nSample Values: 8,3 = 56\n")
    if type(x) == str or type(y) == str:
        raise ValueError("String type not supported")

    try:    
        # print('\nBinomial Coefficient:', fac(x) // fac(y) // fac(x - y))
        return (fac(x) // fac(y) // fac(x - y))

        # x = int(input("Enter First Integer: "))
        # y = int(input("Enter Second Integer: "))
    except:
        print("Input type is not an integer or a -ve value")
      
# print(binomial(10 , 2))      

File: JAVASCRIPT/haris/binomial-coefficient.js
function binomial(n,k) {
<<<<<<< HEAD
    // n = 0;
    // k = 0;
=======
>>>>>>> b6d6148740d94d4063dce8d0399682a9ea55e255
    // var n = prompt("Enter first value: ");
    // var k = prompt("Enter second value: ");
    var coeff = 1;
    for (var x = n - k + 1; x <= n; x++) coeff *= x;
    for (x = 1; x <= k; x++) coeff /= x;
    return coeff;
}

//document.getElementById("bino").innerHTML = binomial();
